pkg/controller/manager/setup.go

```diff
@@ -4,9 +4,11 @@ import (
 	"github.com/harvester/harvester-network-controller/pkg/config"
 	"github.com/harvester/harvester-network-controller/pkg/controller/manager/clusternetwork"
 	"github.com/harvester/harvester-network-controller/pkg/controller/manager/node"
+	"github.com/harvester/harvester-network-controller/pkg/controller/manager/vip"
 )
 
 var RegisterFuncList = []config.RegisterFunc{
 	clusternetwork.Register,
 	node.Register,
+	vip.Register,
```

Comment:
prefer to add a common option to either enable or disable the VIP integration via https://github.com/harvester/network-controller-harvester/blob/master/main.go#L33